src/Model/CompositeTypeGo.cs

```diff
@@ -261,6 +261,15 @@ public virtual string Fields()
                 {
                     indented.Append($"{property.Name} - {property.Documentation}".ToCommentBlock());
                 }
+                if (property.Deprecated)
+                {
+                    var message = "This property has been deprecated.";
+                    if (!string.IsNullOrWhiteSpace(property.DeprecationMessage))
+                    {
+                        message = property.DeprecationMessage;
+                    }
+                    indented.Append($"// Deprecated: {message}\n");

```

Comment:
> Deprecated: {message}\n" [](start = 40, length = 25)

is this going on top of the property doc or on the bottom ? if its on the top its ok but if its on the bottom the previous one needs a  (// ) line I guess.

Reply:
It's on the bottom, will fix.